backend.checkstyle.xml

```diff
@@ -87,7 +87,7 @@
             <property name="max" value="40"/>
         </module>
         <module name="ParameterNumber">
-            <property name="max" value="4"/>
+            <property name="max" value="5"/>
```

Comment:
Can we change it once to `100` and forget about it? :)

I think such changes to be discussed...
I personally have no problems having up to `10` parameters. This is enough in 99% cases.

Reply:
I think, as long as each param has different type, its number does not matter. Its getting really a problem if you have 5 string types. But as this is  not checked by this rule anyway, for me its fine to increase if needed.

Reply:
The purpose of this parameter is usually to force you thinking about good design. If your method requires too many parameters, maybe it does to much and should be splitted.
Since we not follow good design habits anyway, then put here `1000` or just delete this rule.

Why we should keep it if we don't want to follow it anyway?
For me the whole checkstyle file is useless, unless all understand what it is about and why these rules are there... Applying it "because everyone does" is possible, but then stick to the rules and trust the people who created them.

Reply:
removed it